### PR TITLE
Image: adds kyverno and its related images v1.10.0

### PR DESCRIPTION
**Pull request by mritunjaysharma394**
This PR adds kyverno and its related images

Fixes:

Related: https://github.com/wolfi-dev/os/pull/2308

### Pre-review Checklist

- [ ] **IMPORTANT: 'image-request' tag has been applied if this PR is adding any images, including new versions or variants**

#### For new image PRs only

If you have an apko.yaml file in this PR you need to follow this checklist, otherwise feel free to remove.
- [x] Include tests, sufficient enough that you would trust this image running in production.
- [x] The version included is the latest GA version of the software
- [x] The latest tag points to the newest stable version
- [ ] There is a dev tag available that includes a shell and apk tools (by depending on 'wolfi-base')
- [x] The image runs as `nonroot` and GID/UID are set to 65532
  - [ ] Alternatively the username and GID/UID may be a commonly used one from the ecosystem e.g: postgres
- [ ] ENTRYPOINT
  - [ ] For applications/servers/utilities call main program with no arguments e.g. [redis-server]
  - [ ] For base images leave empty
  - [ ] For dev variants set to entrypoint script that falls back to system
- [ ] CMD:
  - [ ] For server applications give arguments to start in daemon mode (may be empty)
  - [ ] For utilities/tooling bring up help e.g. `–help`
  - [ ] For base images with a shell, call it e.g. [/bin/sh]
- [ ] Consider where and how the image deviates from popular alternatives. Is there a good reason and is it documented?
- [x] Add annotations e.g:
```
annotations:
  "org.opencontainers.image.authors": "Chainguard Team https://www.chainguard.dev/"
  "org.opencontainers.image.url": https://edu.chainguard.dev/chainguard/chainguard-images/reference/busybox/ # use the academy site here
  "org.opencontainers.image.source": https://github.com/chainguard-images/images/tree/main/images/bazel # use github here
```
- [ ] Check if environment variables are needed e.g. to set data locations
- [ ] Ensure the image responds to SIGTERM
  - [ ] `docker kill $(docker run -d --rm cgr.dev/chainguard/nginx)`
- [ ] Documentation. Let's make this excellent. Include usage example.
- [ ] Error logs write to stderr and normal logs to stdout. DO NOT write to file.

### CODE REVIEW

**Comment by mritunjaysharma394** [comment]
Oh okay, I think I missed to make the script executable `chmod +x images/superimg/test.sh`  and that's the reason for permission denied in actions failure, just doing it :see_no_evil:

**Comment by tuananh** [comment]
we can do helm install test instead of just startup, like this https://github.com/chainguard-images/images/blob/13c4977ba81b5f17e0534af63f1fdc7b2bcc7149/images/trust-manager/tests/02-helm-install.sh#L26

**Comment by tuananh** [comment]
> ( I need to add two more - background-controller, reports-controller, doing that right away too), will it be okay to have same tests repeating for all the related images with all kyverno and its related images being used in one go?

im not sure about that either. but can you just change only the main one? `repository: kyverno/kyverno`

https://github.com/kyverno/kyverno/blob/7f6fb240579ff6b2961c1ff712c003b298a84dcb/charts/kyverno/values.yaml#LL706C7-L706C34

**Comment by mritunjaysharma394** [comment]
Thanks a lot @tuananh for that great feedback, have added helm install tests for all respective images, let me know how it looks and how the tests show up now, thanks :))

**Review by imjasonh — DISMISSED** [review]
This looks great! 🎉

**Comment by mritunjaysharma394** [comment]
Thanks @imjasonh for the review and approval, I think the tests are failing possibly due to a failed grep of the error, I wanted to know that now that we have the better helm-install tests, should we remove the startup tests altogether that were designed to show error due to not running in k8s? 

Thanks again :heart:

**Comment by imjasonh** [comment]
I think the issue might be that those logs are going to `stderr`, and `grep` is expecting to find them on `stdout`. You can try [redirecting stderr to stdout](https://linuxize.com/post/bash-redirect-stderr-stdout/) before grepping, to see if that helps.

```
docker run $IMAGE_NAME 2>&1 | grep "KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT"`
```

**Comment by mritunjaysharma394** [comment]
Thanks @imjasonh but as far as I remember, I think I am doing something similar only here right? Or am I missing out on something here? Thanks again! 
 https://github.com/chainguard-images/images/blob/710b7fd84ed77f9b35ada563c2afec11cca6c851/images/kyverno-reports-controller/tests/01-startup.sh#L11

**Comment by imjasonh** [comment]
> I wanted to know that now that we have the better helm-install tests, should we remove the startup tests altogether that were designed to show error due to not running in k8s?

Yeah let's just do that. The startup test isn't terribly useful, and the helm test is a lot better.

Thanks for sticking with this, and sorry for the slow reviews, I think we're getting close!

**Comment by mritunjaysharma394** [comment]
Absolutely no worries at all @imjasonh, i totally understand and thanks a lot for that feedback, just did it, keeping with helm tests only for now :))

File: images/kyverno-background-controller/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!--monopod:start-->
+# kyverno-background-controller
+| | |
+| - | - |
+| **Status** | stable |
+| **OCI Reference** | `cgr.dev/chainguard/kyverno-background-controller` |
+| **Variants/Tags** | ![](https://storage.googleapis.com/chainguard-images-build-outputs/summary/kyverno-background-controller.svg) |
```
Comment (imjasonh) [inline images/kyverno-background-controller/README.md:7]: We made some changes to `monopod readme` to remove these lines. Can you update your monopod (`$(cd monopod && go install)`) and regenerate these?

File: README.md [hunk 2]
```diff
@@ -81,6 +81,7 @@
...
 | [kustomize-controller](./images/kustomize-controller) | `cgr.dev/chainguard/kustomize-controller` | experimental | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/kustomize-controller.build.status.latest-dev.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/kustomize-controller:latest-dev)<br/>[![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/kustomize-controller.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/kustomize-controller:latest) |
+| [kyverno-cli](./images/kyverno-cli) | `cgr.dev/chainguard/kyverno-cli` | stable | [![](https://storage.googleapis.com/chainguard-images-build-outputs/badges/kyverno-cli.build.status.latest.svg)](https://registry-ui.chainguard.app/?image=cgr.dev/chainguard/kyverno-cli:latest) |
```
Comment (jdolitsky) [inline README.md:84]: unclear right now why only this one showing up in the readme

Comment (mritunjaysharma394) [reply README.md:84]: Yes, have been trying to figure it out but not sure what could be the reason too, I looked at the code base of the monopod and nothing looked out of the blue yet, I used the most recent build of it and yet I only get kyverno-cli in README

Comment (mritunjaysharma394) [reply README.md:84]: I think this is mostly non-blocking and as per my discussion with @imjasonh on Slack, we can tackle this separately and add it later, if that sounds great!

**Review by imjasonh — APPROVED** [review]

**Pull request merged**